packages/amplication-server/src/core/build/build.service.ts

```diff
@@ -454,15 +454,18 @@ export class BuildService {
         try {
           await this.actionService.logInfo(step, PUSH_TO_GITHUB_STEP_START_LOG);
 
-          const smartGitSyncEntitlement =
-            await this.billingService.getBooleanEntitlement(
-              project.workspaceId,
-              BillingFeature.SmartGitSync
-            );
-
-          const pullRequestMode = smartGitSyncEntitlement.hasAccess
-            ? EnumPullRequestMode.Accumulative
-            : EnumPullRequestMode.Basic;
+          let pullRequestMode = EnumPullRequestMode.Accumulative;
```

Comment:
I think the default suppose to be Basic, @EugeneTseitlin Please have a look here

Reply:
usually billing is disabled for local env, so I think that we can use `Accumulative` mode
Or we can set that mode in `.env` variable

Reply:
> I think the default suppose to be Basic, @EugeneTseitlin Please have a look here

Sure, it has to be basic by default.